test/local/routes/sms.js

```diff
@@ -589,3 +589,103 @@ describe('/sms/status with disabled geo-ip lookup', () => {
   })
 })
 
+describe('/sms/status with query param and enabled geo-ip lookup', () => {
```

Comment:
Is there a case to be made for a test that's `with query param that's not enabled`? For example, if some goon queries the endpoint with `country=AU` will it return `ok: false`?